sycl/doc/GetStartedGuide.md

```diff
@@ -55,55 +60,49 @@ Open a developer command prompt using one of two methods:
 ```bat
 set DPCPP_HOME=%USERPROFILE%\sycl_workspace
 mkdir %DPCPP_HOME%
+cd %DPCPP_HOME%
+
+git clone https://github.com/intel/llvm -b sycl
+mkdir %DPCPP_HOME%\build
+cd %DPCPP_HOME%\build
 ```
 
 # Build DPC++ toolchain
 
-**Linux**
-```bash
-cd $DPCPP_HOME
-git clone https://github.com/intel/llvm -b sycl
-mkdir $DPCPP_HOME/build
-cd $DPCPP_HOME/build
+The easiest way to get started is to use the buildbot [configure](../../buildbot/configure.py)
+and [compile](../../buildbot/configure.py) scripts.
 
-cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" \
--DLLVM_EXTERNAL_PROJECTS="llvm-spirv;sycl" \
--DLLVM_ENABLE_PROJECTS="clang;llvm-spirv;sycl" \
--DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$DPCPP_HOME/llvm/sycl \
--DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$DPCPP_HOME/llvm/llvm-spirv \
-$DPCPP_HOME/llvm/llvm
+In case you want to configure Cmake manually the up-to-date reference for variables is in these files.
 
-make -j`nproc` sycl-toolchain
+**Linux**
+
+```bash
+python $DPCPP_HOME/llvm/buildbot/configure.py -s $DPCPP_HOME/llvm -o $DPCPP_HOME/build -t release
+python $DPCPP_HOME/llvm/buildbot/compile.py -s $DPCPP_HOME/llvm -o $DPCPP_HOME/build
 ```
 
-**Windows (64-bit)**
+**Windows**
+
 ```bat
-cd %DPCPP_HOME%
-git clone https://github.com/intel/llvm -b sycl
-mkdir %DPCPP_HOME%\build
-cd %DPCPP_HOME%\build
+python %DPCPP_HOME%\llvm\buildbot\configure.py -s %DPCPP_HOME%\llvm -o %DPCPP_HOME%\build -t release
+python %DPCPP_HOME%\llvm\buildbot\compile.py -s %DPCPP_HOME%\llvm -o %DPCPP_HOME%\build
```

Comment:
Does this need a particular python version? Does it work with python3?

Reply:
i never tried on windows, but in my linux python3 is default and works.

Reply:
@Ruyk @hiaselhans I don't see anything version-specific in those scripts. I believe both versions are supported. And they definitely work with Python 3, as it's part of CI pipeline.